find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
+  if max_index == 0
+    puts "Array is empty"
+  else
+    max = array[0]
+    index = 0
+    while index < max_index
+      if max < array[index]
+        max = array[index]
+      end
```

Comment:
What do you think about drying this code up a little?

```suggestion
      max = array[index] if max < array[index]
```

Reply:
ah, I see what you mean. I like the only line code better. Thank you it's efficient .